integration/localnet/bootstrap.go

```diff
@@ -58,6 +61,9 @@ func init() {
 	flag.IntVar(&verificationCount, "verification", DefaultVerificationCount, "number of verification nodes")
 	flag.IntVar(&accessCount, "access", DefaultAccessCount, "number of access nodes")
 	flag.UintVar(&nClusters, "nclusters", DefaultNClusters, "number of collector clusters")
+	flag.Uint64Var(&numViewsEpoch, "epoch-length", 0, "number of views in epoch")
+	flag.Uint64Var(&numViewsInStakingPhase, "epoch-staking-phase-length", 0, "number of views in epoch staking phase")
+	flag.Uint64Var(&numViewsInDKGPhase, "epoch-dkg-phase-length", 0, "number of views in epoch dkg phase")
```

Comment:
Is `numViewsEpoch` supposed to be equal to `numViewsInStakingPhase` plus `numViewsInStakingPhase`? Or at least greater than or equal? Can we add some validation for this and return an error if the arguments are invalid?

Reply:
It should be validated in the smart contract. (It wasn't in the case of the constructor, though, so I've made that change in https://github.com/onflow/flow-core-contracts/pull/217)